Minesweeper/Minesweeper/Entities/MineField.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  MineField.swift
+//  Minesweeper
+//
+//  Created by Dorian Grolaux on 19/02/2018.
+//
+
+import Foundation
+
+typealias Coordinates = (x: Int, y: Int)
+
+class MineField {
+    private let cells: [[Cell]]!
+    
+    init(cells: [[Cell]]) {
```

Comment:
Documentation of when we'd use each of these initialisers would be useful. I'm assuming this one is for saved games?